### PR TITLE
INTYGFV-15510: Fix issues with icons, input fields and SelectMultiple

### PR DESCRIPTION
**Pull request by SamuelTilly**
This aims to fix

1. Checkbox icons are not re-rendered correctly inside `SelectMultiple` dropdown
2. `SelectMultiple` does not respect viewbox boundaries.
3. Checkboxes inside `SelectMultiple` should gain focus when opened.
4. Wrong font size for input fields
5. Update `ids-core` to 2.6.0

### CODE REVIEW

File: apps/rehabstod/src/components/Form/NumberInput.tsx
Comment (SamuelTilly) [inline apps/rehabstod/src/components/Form/NumberInput.tsx:33]: `IDSInput` has currently multiple styling issues, border for error input is not rendered correctly. Full-width is not respected. Renamed "isRange" to "inline" to describe layout option.

File: apps/rehabstod/src/components/Form/NumberInput.tsx
Comment (SamuelTilly) [inline apps/rehabstod/src/components/Form/NumberInput.tsx:29]: `className` on reusable components is problematic when refactoring later. It's difficult to know how a component is used and looks in different places when it's allowed to be changed with styling.

Some guideline
- Make sure a component always uses the full width and wrap with an element that constrains the component to fit within the layout.
- Add flags that describes the different states of a component, like with `error` here.
- Add types or styles to describe different styles, look at `DynamicLink` for example.

File: apps/rehabstod/src/components/Form/SelectMultiple.tsx
Comment (mhornfeldt) [inline apps/rehabstod/src/components/Form/SelectMultiple.tsx:26]: We set the text to sm during a UX-meeting so that the lists wouldn't be to large.

Comment (SamuelTilly) [reply apps/rehabstod/src/components/Form/SelectMultiple.tsx:26]: This makes it different size from the NumberInput, looks really strange imo. I'll talk to UX and get a second opinion.

Comment (mhornfeldt) [reply apps/rehabstod/src/components/Form/SelectMultiple.tsx:26]: Otherwise maybe we can introduce a scroll later

File: apps/rehabstod/src/components/SettingsDialogContent/SettingsDialogContent.tsx
Comment (mhornfeldt) [inline apps/rehabstod/src/components/SettingsDialogContent/SettingsDialogContent.tsx:20]: This has introduced a bug where the user can save empty inputs

Comment (SamuelTilly) [reply apps/rehabstod/src/components/SettingsDialogContent/SettingsDialogContent.tsx:20]: Good catch, fixed

**Review by mhornfeldt — APPROVED** [review]

File: apps/rehabstod/src/components/Form/SelectMultiple.tsx
Comment (SamuelTilly) [reply apps/rehabstod/src/components/Form/SelectMultiple.tsx:26]: Talked to UX, it should be 16 for input fields, but it's ok for table cells to have 14px font size. Perhaps there was some confusion here. But we have the go-ahead for removing "text-sm" 😃

**Pull request merged**
